infrastructure/helm-chart/charts/ingres/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Airy Platform networking resources
+name: ingress
+version: 1.0
```

Comment:
Eventually the `Ingress controller` chart should be separate from `Airy Core`, so that people can customize how and if they want to deploy an ingress controller.